code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -514,16 +538,49 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/L, metabolis
 /datum/reagent/peridaxon/overdose_crit_process(mob/living/L, metabolism)
 	L.apply_damages(2*REM, 6*REM, 6*REM)
 
+
+/datum/reagent/medicine/peridaxon_plus
+	name = "Peridaxon"
+	description = "Used to repair internal organs. Significant toxic and stun side effects."
+	color = "#C845DC"
+	overdose_threshold = REAGENTS_OVERDOSE/30
+	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL/25
+	custom_metabolism = REAGENTS_METABOLISM * 0.25
+	scannable = TRUE
+
+/datum/reagent/medicine/peridaxon_plus/on_mob_life(mob/living/L, metabolism)
+	L.apply_damage(10*REM, TOX) //50 toxin/unit
+	L.adjustHalLoss(300) //You do not get up while this is in you. At all.
+	if(!ishuman(L))
+		return ..()
+	var/mob/living/carbon/human/H = L
+	for(var/datum/internal_organ/I in H.internal_organs)
+		I.heal_organ_damage(4*REM) // 2/tick
+	return ..()
+
+/datum/reagent/medicine/peridaxon_plus/overdose_process(mob/living/L, metabolism)
+	L.apply_damage(20*REM, BRUTE)
+
+/datum/reagent/peridaxon_plus/overdose_crit_process(mob/living/L, metabolism)
+	L.apply_damages(20*REM, BRUTE) //100 damage/unit. Don't OD.
+
+
+
 /datum/reagent/medicine/bicaridine
 	name = "Bicaridine"
-	description = "Bicaridine is an analgesic medication and can be used to treat blunt trauma."
+	description = "Bicaridine is an analgesic medication and can be used to treat blunt trauma. More effective in higher dosage, but causes slight pain and slowdown."
```

Comment:
It says it's analgesic. I'm not seeing a slowdown effect.

Reply:
There's a pain effect over 5u and a greater one over 15u. I pulled all slowdowns when you mentioned you were porting a new movement system.

Regardless, I should probably edit that description to reflect reality.

Reply:
Altered.